### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by javaporter** [javaporter/roots-contentful-foundation#9]
I'm seeing this quite often. Even when I'm not actively working on the site it'll pop up and roots will crash.

**Comment by jenius** [comment]
Segfault is a system-level memory issue usually. I haven't seen this one before. Do you have a reliable way to reproduce the issue?

**Comment by javaporter** [comment]
nope, it's pretty random, but it's equally random on a 5K iMac and a 2015 Macbook. I realize that this is nearly impossible to troubleshoot, I just made an issue for it for my own notes really. It happens about once an hour.

**Comment by javaporter** [comment]
just logging for grins

```
pk@ roots-contentful-foundation (master) $ roots watch
compiling... done!
compiling... done!
compiling... done!
compiling... compiling... done!
done!
compiling... Segmentation fault: 11
pk@ roots-contentful-foundation (master) $ 
```
